src/main/resources/config/liquibase/changelog/20240902175045_changelog.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240902132940" author="cremer">
+        <createTable tableName="faq">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="question_title" type="varchar(2000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="question_answer" type="varchar(2000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="course_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="faq_state" type="varchar(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_by" type="varchar(45)"/>
+            <column name="last_modified_by" type="varchar(45)"/>
+            <column name="last_modified_date" type="datetime(3)"/>
+            <column name="created_date" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <createTable tableName="faq_categories">
+            <column name="faq_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="categories" type="varchar(50)"/>
```

Comment:
Tried to stay consistent here. Since this property is used in the category logic and i want to refactor it to use a distinct service, i need to stay consistent with the field name from exercise_categories here